tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Checkboxes', () => {
+    test.beforeEach(async ({page}) => {
+        await page.getByRole('button', {name: " Veterinarians"}).click()
+        await page.getByRole('link', {name: " All"}).click() //1. Select the VETERINARIANS menu item in the nav bar, then select "All"
+    })
+
+    test('validate selected specialties', async ({page}) => {
+        const vetDoctorTableHeader = page.locator('app-vet-list h2')
+        await expect(vetDoctorTableHeader).toHaveText('Veterinarians') //2. Assert Veterinarians text is displayed above table
+        const vetTableRowDr = page.getByRole('row', {name: " Helen Leary "})
+        await vetTableRowDr.getByRole('button', {name: 'Edit Vet'}).click() //3. Select Helen Leary and click Edit Vet
+        const specialtyDropdown = page.locator('.dropdown-display .selected-specialties') //4. Assert Specialties field is displayed with radiology
+        await expect(specialtyDropdown).toHaveText('radiology')
+        const speacialtiesDropdownArrow = page.locator('.dropdown-arrow')
+        await speacialtiesDropdownArrow.click() //5. Click on dropdown
+        expect(await page.getByRole('checkbox', {name: 'radiology'}).isChecked()) //6. Assert radiology is checked
+        expect(await page.getByRole('checkbox', {name: 'surgery'}).isChecked()).toBeFalsy() //7. Assert that surgery and dentistrty are unchecked
+        expect(await page.getByRole('checkbox', {name: 'dentistry'}).isChecked()).toBeFalsy()
+
+        await page.getByRole('checkbox', {name: 'radiology'}).uncheck({force: true}) //8. Check surgery and uncheck radiology
+        await page.getByRole('checkbox', {name: 'surgery'}).check({force: true})
+        await expect(specialtyDropdown).toHaveText('surgery') //9. Assert specialties field displays surgery
+
+        await page.getByRole('checkbox', {name: 'dentistry'}).check({force: true}) //10.Check the dentistry checkbox
+        await expect(specialtyDropdown).toHaveText('surgery, dentistry') //11. Assert that specialties field displays surgery and dentistry
+        await speacialtiesDropdownArrow.click()
```

Comment:
It looks like this line is not needed here.

Reply:
Yes, I know the line was not needed but I added it so that the specialties box would close. But when I combine the previous lines then closing the specialties dropdown no longer works. Removed